DashAI/front/package.json

```diff
@@ -2,18 +2,20 @@
   "name": "dashai-frontend",
   "version": "0.1.0",
   "private": true,
+  "packageManager": "yarn@3.5.0",
   "dependencies": {
-    "@mui/material": "^5.11.13",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
```

Comment:
Are we adding emotion? I remember @maximi4321 talked about using styled-components. Please confirm it before adding another CSS-in-JS library

Reply:
As discussed, we will gradually replace styled components with emotion.